bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZZGTo4L_4f_NLO/ZZGTO4L_4f_NLO_run_card.dat

```diff
@@ -0,0 +1,144 @@
+ .true.   = reweight_scale   ! reweight to get scale dependence
+  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
+  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
+  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
+  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+$DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
+#***********************************************************************
+# Merging - WARNING! Applies merging only at the hard-event level.     *
+# After showering an MLM-type merging should be applied as well.       *
+# See http://amcatnlo.cern.ch/FxFx_merging.htm for more details.       *
+#***********************************************************************
+ 0        = ickkw            ! 0 no merging, 3 FxFx merging
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+ 30  = bwcutoff
+#***********************************************************************
+# Cuts on the jets                                                     *
+# Jet clustering is performed by FastJet.
+# When matching to a parton shower, these generation cuts should be    *
+# considerably softer than the analysis cuts.                          *
+# (more specific cuts can be specified in SubProcesses/cuts.f)         *
+#***********************************************************************
+   1  = jetalgo   ! FastJet jet algorithm (1=kT, 0=C/A, -1=anti-kT)
+ 0.7  = jetradius ! The radius parameter for the jet algorithm
+  10  = ptj       ! Min jet transverse momentum
+  -1  = etaj      ! Max jet abs(pseudo-rap) (a value .lt.0 means no cut)
+#***********************************************************************
+# Cuts on the charged leptons (e+, e-, mu+, mu-, tau+ and tau-)        *
+# (more specific gen cuts can be specified in SubProcesses/cuts.f)     *
+#***********************************************************************
+   3  = ptl     ! Min lepton transverse momentum
+  -1  = etal    ! Max lepton abs(pseudo-rap) (a value .lt.0 means no cut)
+   0  = drll    ! Min distance between opposite sign lepton pairs
+   0  = drll_sf ! Min distance between opp. sign same-flavor lepton pairs
+   1  = mll     ! Min inv. mass of all opposite sign lepton pairs
+   1  = mll_sf  ! Min inv. mass of all opp. sign same-flavor lepton pairs
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+#***********************************************************************
+  8  = ptgmin    ! Min photon transverse momentum
```

Comment:
Your RECO level gamma pt cut will be ~ 8GeV?

Reply:
yes, it seems that the ptgmin cut influence more to our analysis showing [here](https://indico.cern.ch/event/1013238/contributions/4252831/attachments/2219928/3759126/mmm_ptl10.png),  so we think it should more inclusive.

Reply:
so the cut on your photons is > 10 GeV?

Reply:
Yes, primary

Reply:
Sorry I meant it as a RECO level photon pT cut.
As long as the GEN level cuts are safely inclusive (+ not really wasting a lot of phase space : cases like if you give mll>70 GeV in RECO level but give >10 GeV cut in GEN level which would yield a huge waste) the cut values are okay.

Reply:
> Just one minor question,
> Is there a reason to enhance the bwcutoff to 30? The default should be 15 so I am just curious.

Ops, I forget, there is no z, I change back to 15.

Reply:
and, any other suggestion about runcard?